### PR TITLE
Update sl container for sl6 to current (build-date=20190603)

### PR DESCRIPTION
**Pull request by sl-team**

### CODE REVIEW

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/sl_6/Dockerfile b/sl_6/Dockerfile
index 78752f1..1e142c1 100644
--- a/sl_6/Dockerfile
+++ b/sl_6/Dockerfile
@@ -2,7 +2,7 @@
 FROM scratch
 ADD sl-6-docker.tar.xz /
 
-LABEL name="SL6 Base Image" vendor="Scientific Linux" build-date="20190501"
+LABEL name="SL6 Base Image" vendor="Scientific Linux" build-date="20190603"
 
 CMD ["/bin/bash"]
 
diff --git a/sl_6/sl-6-docker.tar.xz b/sl_6/sl-6-docker.tar.xz
index 428390a..9a70fe0 100644
Binary files a/sl_6/sl-6-docker.tar.xz and b/sl_6/sl-6-docker.tar.xz differ
```

</details>

**Comment by yosifkit** [comment]
Build test of #6009; d8731d0f; `amd64` (`sl`):

```console

$ bashbrew build sl:7
Using bashbrew/cache:f802b7a22f7e9c79219c3c07857e1954d356e311ddd26b26c10b3310efcab230 (sl:7)
Tagging sl:7
Tagging sl:latest

$ test/run.sh sl:7
testing sl:7
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...warning: garbage password detected for 'root': 'locked'
passed
	'override-cmd' [4/4]...passed


$ bashbrew build sl:6
Building bashbrew/cache:9ba8a888538481236a6b971f1045a8401b13f230dd1b139bf5fdc4e4274ea5d3 (sl:6)
Tagging sl:6

$ test/run.sh sl:6
testing sl:6
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...warning: garbage password detected for 'root': 'locked'
passed
	'override-cmd' [4/4]...passed

```

**Comment by yosifkit** [comment]
It seems we have been less than diligent in checking history and the branches for 6 and 7 have accumulated quite a few old tarballs in the history. Can you please rebase those branches to only have a single commit which touches the large tarballs to help save on cloning bandwidth/time/storage?

https://github.com/docker-library/official-images/pull/4804#issuecomment-418509635
https://github.com/docker-library/official-images/pull/4608#issuecomment-406023313

**Comment by jcpunk** [comment]
should be fixed now

**Comment by yosifkit** [comment]
Thanks @jcpunk! Those branches look great.

File: library/sl [hunk 1]
```diff
@@ -15,4 +15,4 @@ Tags: 6
 Architectures: amd64
 GitFetch: refs/heads/6
 Directory: sl6
-GitCommit: 17babe8c23b74cc51ec4bce25d789157650aa8e9
+GitCommit: 042f97be7cece26d56892893bd80a1e48d7b0cd7
```
Comment (yosifkit) [inline library/sl]:
```suggestion
GitCommit: 5288d401c76c12025e0495265067ad6a912b59ce
```

Update to reflect the new commit id in https://github.com/scientificlinux/sl-docker/commits/6

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/sl_6/Dockerfile b/sl_6/Dockerfile
index 78752f1..1e142c1 100644
--- a/sl_6/Dockerfile
+++ b/sl_6/Dockerfile
@@ -2,7 +2,7 @@
 FROM scratch
 ADD sl-6-docker.tar.xz /
 
-LABEL name="SL6 Base Image" vendor="Scientific Linux" build-date="20190501"
+LABEL name="SL6 Base Image" vendor="Scientific Linux" build-date="20190603"
 
 CMD ["/bin/bash"]
 
diff --git a/sl_6/sl-6-docker.tar.xz b/sl_6/sl-6-docker.tar.xz
index 428390a..9a70fe0 100644
Binary files a/sl_6/sl-6-docker.tar.xz and b/sl_6/sl-6-docker.tar.xz differ
```

</details>

Build test of #6009; 12432c83; `amd64` (`sl`):

```console

$ bashbrew build sl:7
Using bashbrew/cache:f802b7a22f7e9c79219c3c07857e1954d356e311ddd26b26c10b3310efcab230 (sl:7)
Tagging sl:7
Tagging sl:latest

$ test/run.sh sl:7
testing sl:7
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...warning: garbage password detected for 'root': 'locked'
passed
	'override-cmd' [4/4]...passed


$ bashbrew build sl:6
Building bashbrew/cache:2a3d84f537237ebc4803c2176cb1f550e06cc65a6fa2505a6f2d90a206b47f4f (sl:6)
Tagging sl:6

$ test/run.sh sl:6
testing sl:6
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...warning: garbage password detected for 'root': 'locked'
passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**